packages/apps-v5/src/commands/ps/type.js

```diff
@@ -3,7 +3,7 @@
 let cli = require('heroku-cli-util')
 const { sortBy, compact } = require('lodash')
 
-const costs = { 'Free': 0, 'Hobby': 7, 'Standard-1X': 25, 'Standard-2X': 50, 'Performance-M': 250, 'Performance': 500, 'Performance-L': 500, '1X': 36, '2X': 72, 'PX': 576 }
+const costs = { 'Eco': 5, 'Basic': 7, 'Standard-1X': 25, 'Standard-2X': 50, 'Performance-M': 250, 'Performance': 500, 'Performance-L': 500, '1X': 36, '2X': 72, 'PX': 576 }
```

Comment:
This is an interesting one. Tested this command and it appears that this code will tell you your monthly dyno costs. Eco costs $5/month across ALL of your eco dynos. Also, it will only charge if you are subscribed to eco. 

```
=== Dyno Types
type     size         qty  cost/mo
───────  ───────────  ───  ───────
release  Standard-1X  0    0
web      Standard-1X  1    25
=== Dyno Totals
type         total
───────────  ─────
Standard-1X  1
```

If they are not subscribed to eco, their eco dynos will be scaled to 0. So that makes it easy to tell whether or not they are subscribed. If they have at least 1 dyno, then they are definitely subscribed.

IMO we should set Eco to $0 as the cost. And then, if they have at least 1 eco dyno, we should print a line letting the customer know that they are being charged $5 per month across all of their eco dynos in addition to the other dyno costs. That would be most accurate. I'd refer to @arianaescobar @SandyPantsLai for copy on that.

Reply:
`$5 (flat monthly fee, shared across all Eco dynos)`?

Reply:
and @zwhitfield3 let's add $ signs for the cost as well to make it more obvious what currency we are talking about here.